tests/integration/standard/test_query.py

```diff
@@ -386,6 +385,21 @@ def test_no_parameters_many_times(self):
             self.test_no_parameters()
             self.session.execute("TRUNCATE test3rf.test")
 
+    def test_unicode(self):
+        ddl = '''
+            CREATE TABLE test3rf.testtext (
```

Comment:
should this be using any of the base keyspace or table properties since the test is now inheriting the common base?